packages/core/src/components/navbar/navbarGroup.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the terms of the LICENSE file distributed with this project.
+ */
+
+import * as classNames from "classnames";
+import * as PureRender from "pure-render-decorator";
+import * as React from "react";
+import * as Classes from "../../common/classes";
+import { IProps } from "../../common/props";
+
+export interface INavbarGroupProps extends IProps {
+    /**
+     * The side of the navbar on which the group should appear.
+     * @default "left"
+     */
+    align?: "left" | "right";
+}
+
+@PureRender
+export class NavbarGroup extends React.Component<INavbarGroupProps & { children?: React.ReactNode }, {}> {
+    public static defaultProps: INavbarGroupProps = {
+        align: "left",
```

Comment:
`right` seems like a more decent default yeah? i.e. the heading on the left and rest on the right

Reply:
`left` seems more reasonable to me because it comes first, both alphabetically and visually.

Reply:
Yeah I think `left` is more intuitive personally. Another example elsewhere in the design system: `pt-align-right` needs to be explicitly applied to align something to the right; otherwise, the element in question will align to the left by default.